sql/parse/parse_test.go

```diff
@@ -1636,28 +1636,28 @@ var fixtures = map[string]sql.Node{
-	"SHOW CREATE VIEW `mytable`":            plan.NewShowCreateTable("", nil, plan.NewUnresolvedTable("mytable", ""), true),
-	"SHOW CREATE VIEW mytable":              plan.NewShowCreateTable("", nil, plan.NewUnresolvedTable("mytable", ""), true),
-	"SHOW CREATE VIEW mydb.`mytable`":       plan.NewShowCreateTable("mydb", nil, plan.NewUnresolvedTable("mytable", "mydb"), true),
-	"SHOW CREATE VIEW `mydb`.mytable":       plan.NewShowCreateTable("mydb", nil, plan.NewUnresolvedTable("mytable", "mydb"), true),
-	"SHOW CREATE VIEW `mydb`.`mytable`":     plan.NewShowCreateTable("mydb", nil, plan.NewUnresolvedTable("mytable", "mydb"), true),
-	"SHOW CREATE VIEW `my.table`":           plan.NewShowCreateTable("", nil, plan.NewUnresolvedTable("my.table", ""), true),
-	"SHOW CREATE VIEW `my.db`.`my.table`":   plan.NewShowCreateTable("my.db", nil, plan.NewUnresolvedTable("my.table", "my.db"), true),
-	"SHOW CREATE VIEW `my``table`":          plan.NewShowCreateTable("", nil, plan.NewUnresolvedTable("my`table", ""), true),
-	"SHOW CREATE VIEW `my``db`.`my``table`": plan.NewShowCreateTable("my`db", nil, plan.NewUnresolvedTable("my`table", "my`db"), true),
-	"SHOW CREATE VIEW ````":                 plan.NewShowCreateTable("", nil, plan.NewUnresolvedTable("`", ""), true),
-	"SHOW CREATE VIEW `.`":                  plan.NewShowCreateTable("", nil, plan.NewUnresolvedTable(".", ""), true),
+	"SHOW CREATE TABLE `mytable`":            plan.NewShowCreateTable(plan.NewUnresolvedTable("mytable", ""), false),
+	"SHOW CREATE TABLE mytable":              plan.NewShowCreateTable(plan.NewUnresolvedTable("mytable", ""), false),
+	"SHOW CREATE TABLE mydb.`mytable`":       plan.NewShowCreateTable(plan.NewUnresolvedTable("mytable", "mydb"), false),
+	"SHOW CREATE TABLE `mydb`.mytable":       plan.NewShowCreateTable(plan.NewUnresolvedTable("mytable", "mydb"), false),
+	"SHOW CREATE TABLE `mydb`.`mytable`":     plan.NewShowCreateTable(plan.NewUnresolvedTable("mytable", "mydb"), false),
+	"SHOW CREATE TABLE `my.table`":           plan.NewShowCreateTable(plan.NewUnresolvedTable("my.table", ""), false),
+	"SHOW CREATE TABLE `my.db`.`my.table`":   plan.NewShowCreateTable(plan.NewUnresolvedTable("my.table", "my.db"), false),
+	"SHOW CREATE TABLE `my``table`":          plan.NewShowCreateTable(plan.NewUnresolvedTable("my`table", ""), false),
+	"SHOW CREATE TABLE `my``db`.`my``table`": plan.NewShowCreateTable(plan.NewUnresolvedTable("my`table", "my`db"), false),
+	"SHOW CREATE TABLE ````":                 plan.NewShowCreateTable(plan.NewUnresolvedTable("`", ""), false),
+	"SHOW CREATE TABLE `.`":                  plan.NewShowCreateTable(plan.NewUnresolvedTable(".", ""), false),
+	"SHOW CREATE VIEW `mytable`":            plan.NewShowCreateTable( plan.NewUnresolvedTable("mytable", ""), true),
+	"SHOW CREATE VIEW mytable":              plan.NewShowCreateTable( plan.NewUnresolvedTable("mytable", ""), true),
```

Comment:
`plan.NewShowCreateTable( plan.`
I don't have OCD but the random space lol. I can't see a pattern to it either XD

Reply:
Done.